pylint_odoo/checkers/modules_odoo.py

```diff
@@ -398,6 +404,14 @@ def visit_import(self, node):
             if isinstance(node.scope(), astroid.Module):
                 self._check_imported_packages(node, name)
 
+    @utils.check_messages('except-pass')
+    def visit_tryexcept(self, node):
+        """Visit block try except"""
+        for handler in node.handlers:
+            for body in handler.body:
```

Comment:
Just verify the first item because if we have other sentence then is fine.
I mean,

```
try:
   sentence
except:
   print("qp2")
   pass
```

This is a bad practice too but is other matter